x-pack/plugins/security_solution/common/api/timeline/pinned_events/pinned_events_route.ts

```diff
@@ -5,26 +5,14 @@
  * 2.0.
  */
 
-/* eslint-disable @typescript-eslint/no-empty-interface */
-
 import * as runtimeTypes from 'io-ts';
 import { unionWithNullType } from '../../../utility_types';
 
-export const pinnedEventIds = unionWithNullType(runtimeTypes.array(runtimeTypes.string));
-export const persistPinnedEventSchema = runtimeTypes.intersection([
-  runtimeTypes.type({
-    eventId: runtimeTypes.string,
-    timelineId: runtimeTypes.string,
-  }),
-  runtimeTypes.partial({
-    pinnedEventId: unionWithNullType(runtimeTypes.string),
-  }),
-]);
-
 /*
- *  Note Types
+ * Pinned Event Types
+ * TODO: remove these when the timeline types are moved to zod
```

Comment:
These runtime types are still used in more nested timeline type checks that will be tackled in another PR.